sdk/webpubsub/Azure.Messaging.WebPubSub/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-07-16)
```

Comment:
I need help to fill in the changelog for this release.

Reply:
- Added support for protocol methods RequestOptions: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md#using-requestoptions-to-customize-behavior
- Added support for API Management
- Bug fixes
- Simplified dependencies.